misk-hibernate/src/main/kotlin/misk/hibernate/ReflectionQueryFactory.kt

```diff
@@ -560,6 +583,37 @@ internal class ReflectionQuery<T : DbEntity<T>>(
         }
       }
 
+      private fun createFetch(
+        errors: MutableList<String>,
+        function: KFunction<*>,
+        result: MutableMap<Method, QueryMethodHandler>,
+        fetch: Fetch
+      ) {
+        // TODO (peckb1) refactor into method instead of copy/paste
+        if (!fetch.path.matches(PATH_PATTERN)) {
+          errors.add("${function.name}() path is not valid: '${fetch.path}'")
+          return
+        }
+
+        // TODO (peckb1) refactor into method instead of copy/paste
```

Comment:
Would we be ok extracting out the common validation as part of this PR, or should it be a separate PR?

Reply:
looking at abstracting out the validation: the common methods never looked … good. 
The code just always seemed uglier and harder to follow.